aptos-move/framework/aptos-framework/sources/object.move

```diff
@@ -67,11 +69,19 @@ module aptos_framework::object {
     /// A shared resource group for storing object resources together in storage.
     struct ObjectGroup { }
 
-    /// Type safe way of designate an object as at this address.
+    /// Type-safe way of designate an object as at this address.
```

Comment:
Don't understand what this comment wants to say. Definitely ObjectId is not type-safe?